attributioncode/validator.go

```diff
@@ -201,3 +215,14 @@ func checkMAC(key, msg, msgMAC []byte) error {
 	}
 	return nil
 }
+
+func fromRTAMO(content string) bool {
+	matched, err := regexp.MatchString(`^rta:`, content)
+	if err != nil {
+		return false
+	}
+	if matched {
+		return true
+	}
+	return false
+}
```

Comment:
This is similar to the [fromRTAMO function in bouncer](https://github.com/mozilla-services/go-bouncer/blob/master/handlers.go#L361-L387) but requires less since the `attribution_code` is already parsed and decoded before getting to this check.